backend/hitas/views/housing_company.py

```diff
@@ -45,6 +49,18 @@
 from hitas.views.utils.serializers import YearMonthSerializer
 
 
+class RegulationStatusViewSet(ViewSet):
+    def list(self, request: Request, *args, **kwargs) -> Response:
+        data = [
+            {
+                "name": regulation_status.value,
```

Comment:
`value` or `name`?

`name` sounds kind of synonymous with `label` to me 🤔

Reply:
I guess we can go with the value-label separation enums use. I'll change it.